pycheribuild/projects/run_fvp.py

```diff
@@ -475,6 +481,14 @@ def setup_config_options(cls, **kwargs):
         cls.smp = cls.add_bool_option("smp", help="Simulate multiple CPU cores in the FVP", default=True)
         cls.force_headless = cls.add_bool_option("force-headless", default=False,
                                                  help="Force headless use of the FVP")
+        cls.fvp_gdbserver = cls.add_bool_option("gdbserver", default=False,
```

Comment:
Maybe this should just use the global `--wait-for-debugger` option (or use that as the default value)?